lib/buffer.js

```diff
@@ -17,6 +17,48 @@ var poolSize, poolOffset, allocPool;
 
 
 binding.setupBufferJS(Buffer.prototype, bindingObj);
+
+const swap16n = Buffer.prototype.swap16;
+const swap32n = Buffer.prototype.swap32;
```

Comment:
Why are we doing this instead of attaching the native implementation to `binding.swap*()`?